Source/relay/TourGuide/Items of the Month/2023/Closed Circuit Pay Phone.ash

```diff
@@ -62,14 +62,22 @@ void IOTMClosedCircuitPayPhoneGenerateTasks(ChecklistEntry [int] task_entries, C
     }
     else if (!state.started) {
         boolean calledRufusToday = get_property_boolean("_shadowAffinityToday");
-        string textColor = calledRufusToday ? "black" : "red";
+        string textColor = calledRufusToday ? "black" : "blue";
         string callRufusMessage = calledRufusToday ? "Optionally call Rufus again for another (turn-taking) quest." : "Haven't called Rufus yet today.";
         rufusQuestDescription.listAppend(HTMLGenerateSpanFont(callRufusMessage, textColor));
         rufusQuestTitle = "Rufus quest doable now";
         rufusQuestPriority = 11;
         whereToAddRufusQuestTile = optional_task_entries;
     }
 
+    string [int][int] shadowBricksTable;
```

Comment:
i like the tooltip approach. my only small note -- should we color the text gray if the user doesn't have them unlocked? cemetery should always be unlocked (well, not always, but they should be able to hit level 7 by EOD day 1) but HC/pyramid definitely can be hard to snag on d1 depending on your routing. i think quest state prefs can tell us availability on those.

Reply:
Yep I'll add these checks 👍

Reply:
Used mafia `can_adventure` for this. I'm in CS aftercore and it's returning false for all of the brick rifts, and true for the ones that are actually accessible like Right Side of the Tracks, so it should work fine without having to do a bunch of quest state tracking ourselves.